users/tests/test_models.py

```diff
@@ -38,7 +38,7 @@ def application():
 
 @pytest.fixture
 def oidc_client():
-    return OIDCClientFactory(response_type='id_token token')
+    return OIDCClientFactory(response_types=['id_token token'])
```

Comment:
Same here: Should this be a list of two strings rather than just one?

Reply:
Same answer.

Reply:
...and this.